examples/exception-handling/retry/retry-5/ResiliencePipelineFactory.cs

```diff
@@ -19,7 +19,6 @@ public ResiliencePipeline GetPipeline( StrategyKind strategyKind )
                             BackoffType = DelayBackoffType.Exponential,
                             MaxRetryAttempts = 10
                         } );
```

Comment:
Inconsistent formatting. The spaces around brackets are kept in some places and removed in others. Check that the code style is set up correctly. See https://github.com/postsharp/postSharp.Engineering.codestyle.

Reply:
I ran `b codestyle pull`, fixed path to CommonStyle, and ran `b codestyle format` (in a separate commit).